.github/workflows/build.yml

```
@@ -19,9 +19,18 @@ jobs:
       # Run `git checkout`
       - uses: actions/checkout@v3
 
-      - name: Retrieve current version from pom
+      - name: Set up JDK 17
+        uses: actions/setup-java@v3
+        with:
+          java-version: '17'
+          distribution: 'temurin'
+
+      - name: Make gradlew executable
+        run: chmod +x ./gradlew
+
+      - name: Retrieve current version from gradle.properties
         run: |
-          echo RELEASE_VERSION=$(cat pom.xml | grep -o -P '(?<=revision\>).*(?=\<\/revision)') >> $GITHUB_ENV
+          echo RELEASE_VERSION=$(./gradlew properties --no-daemon --console=plain -q | grep "^version:" | awk '{print $2}') >> $GITHUB_ENV
```

```suggestion
          echo RELEASE_VERSION=$(./gradlew properties --no-daemon --console=plain -q | grep "^version:" | awk '{print $2}')  | sed "s/-SNAPSHOT//" >> $GITHUB_ENV
```

I forgot to remove the SNAPSHOT part in my example, otherwise RELEASE_VERSION would be `9.1.0-SNAPSHOT`, but we only care about the actual version numbers here.

---

Fixed